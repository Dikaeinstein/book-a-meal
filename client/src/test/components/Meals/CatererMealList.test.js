import React from 'react';
import { shallow } from 'enzyme';
import CatererMealList
  from '../../../components/Meals/CatererMealLIst';

describe('<CatererMealList />', () => {
  const meals = [
    { id: 1, name: 'test meal', price: 4000 },
    { id: 2, name: 'test meal2', price: 4000 },
  ];
  const props = { meals, handleMealUpdate: jest.fn() };
  it('should match snapshot', () => {
    const tree = shallow(<CatererMealList {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
