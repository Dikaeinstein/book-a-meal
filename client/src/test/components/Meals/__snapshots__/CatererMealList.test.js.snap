// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CatererMealList /> should match snapshot 1`] = `
<section
  className="card user-menu text-left"
>
  <Connect(Meal)
    handleMealUpdate={[MockFunction]}
    key="1"
    meal={
      Object {
        "id": 1,
        "name": "test meal",
        "price": 4000,
      }
    }
  />
  <Connect(Meal)
    handleMealUpdate={[MockFunction]}
    key="2"
    meal={
      Object {
        "id": 2,
        "name": "test meal2",
        "price": 4000,
      }
    }
  />
</section>
`;
